addons/escoria-core/game/core-scripts/esc/commands/custom.gd

```diff
@@ -81,6 +81,11 @@ func run(command_params: Array) -> int:
 	var object = escoria.object_manager.get_object(
 		command_params[0]
 	)
+	# Global variables can be substituted into the command arguments by wrapping the global
+	# name in braces.
+	for loop in command_params[3].size():
+		command_params[3][loop] = escoria.logger.replace_globals(command_params[3][loop])
```

Comment:
If we're going to be replacing globals for things other than log-related commands, then I'd like to see `replace_globals` extracted from `logger` and put into a more appropriate place, e.g. `ESCCompiler`. @StraToN would you agree with that location? Or would you prefer it put somewhere else?

Reply:
I totally agree, `replace_globals` will move out of ESCLogger.